### DISCUSSION TITLE
GPU isnt showing up

### DISCUSSION THREAD

**Discussion by Fishman2294** [Rem0o/FanControl.Releases#822] [General]
my GPU fan isn't showing up I updated AMD software and now there's no GPU or fan control for it

**Comment by Rem0o** [comment]
see https://github.com/Rem0o/FanControl.Releases/issues/821

**Reply by Fishman2294** [reply]
would you say trying to downgrade the amd software do you think that would work

**Reply by Rem0o** [reply]
Could should would, only one way to find out.
